3rdParty/iresearch/core/search/disjunction.hpp

```diff
@@ -1388,8 +1390,8 @@ class block_disjunction final
   size_t match_count_;
   cost cost_;
   size_t buf_offset_{}; // offset within a buffer
-  score_buffer_type score_buf_;
-  min_match_buffer_type match_buf_;
+  score_buffer_type score_buf_;     // FIXME EBO
+  min_match_buffer_type match_buf_; // FIXME EBO
```

Comment:
What does `EBO` mean in this context? I guess it refers to some optimization that we can do in the future?

Reply:
"empty base optimization"